common/lib/xmodule/xmodule/js/spec/video/video_caption_spec.js

```diff
@@ -764,11 +765,10 @@
                 Caption.fetchCaption();
 
                 expect(Caption.fetchAvailableTranslations).not.toHaveBeenCalled();
-                expect($.ajaxWithPrefix.calls.mostRecent().args[0].data)
```

Comment:
@burhandodhy why we have to remove this line?

Reply:
@ziafazal `$.ajaxWithPrefix.calls.mostRecent()` have `fetchCaption` functions data before my fix. As now we call `hideClosedCaptions` function opon `fetchCaption` failure `$.ajaxWithPrefix.calls.mostRecent()` now have `hideClosedCaptions` data.